QuestionBot/QuestionBot.UnitTests/Model/StoreTests.cs

```diff
@@ -1,13 +1,77 @@
-﻿using NUnit.Framework;
+﻿using System;
+using System.Linq;
+using NUnit.Framework;
+using System.Collections.Generic;
+using QuestionBot.Model;
 
 namespace QuestionBot.UnitTests.Model {
 
     [TestFixture]
-    public class StoreTests {
+    public class StoreTests
+    {
+        private IStore _storeTest;
+        private IEnumerable<IRecord> _retreivedRecords = new List<IRecord>();
 
+    [SetUp]
+    public void Setup(){
+        _storeTest = new InMemoryStore();
+    }
+        [Test]
+        public void New_question_creates_record()
+        {
+            string question = "What is 1+2?";
+            IRecord questionRecord = _storeTest.CreateRecord(question);
+            Assert.AreEqual(questionRecord.Question, question);
+            Assert.AreNotEqual(questionRecord.TimeAsked, DateTime.MinValue);
+            Assert.AreNotEqual(questionRecord.ID,0);
```

Comment:
Sure thing! In both cases I want to make sure that when I add a new question, the question record isn't being created with the default values for the ID and DateTime, I want to make sure they are being changed.

1. I read about the initial/default value of DateTime, and how to check if the default is being used. A lot of responses said to use a nullable DateTime and you can check if DateTime is null, but I also learned that ["MinValue defines the date and time that is assigned to an uninitialized DateTime variable" ](https://msdn.microsoft.com/en-us/library/system.datetime.minvalue(v=vs.110).aspx) and is equivalent to 00:00:00.0000000 UTC, January 1, 0001 - so I decided to check for this MinValue in an effort to make sure the default DateTime was not being used.

2. I initialize the question ID in InMemoryStore.cs with a value of 0, so the test Assert.AreNotEqual(questionRecord.ID,0) is being used to make sure the ID for the question created is not just the default value.